tensorboard/webapp/runs/views/runs_table/runs_data_table.ts

```diff
@@ -36,32 +36,39 @@ export class RunsDataTable {
   @Input() headers!: ColumnHeader[];
   @Input() data!: TableData[];
   @Input() sortingInfo!: SortingInfo;
+  @Input() experimentIds!: string[];
 
   ColumnHeaderType = ColumnHeaderType;
 
   @Output() sortDataBy = new EventEmitter<SortingInfo>();
   @Output() orderColumns = new EventEmitter<ColumnHeader[]>();
   @Output() onSelectionToggle = new EventEmitter<string>();
   @Output() onAllSelectionToggle = new EventEmitter<string[]>();
+  @Output() onRunColorChange = new EventEmitter<{
+    runId: string;
+    newColor: string;
+  }>();
+
+  beforeColumns = [
+    {
+      name: 'selected',
+      displayName: '',
+      type: ColumnHeaderType.CUSTOM,
+      enabled: true,
+    },
+  ];
+
+  afterColumns = [
+    {
+      name: 'color',
+      displayName: '',
+      type: ColumnHeaderType.COLOR,
+      enabled: true,
+    },
+  ];
 
   getHeaders() {
```

Comment:
You mentioned having an issue with having the color picker open and close immediately because the reference was changing. It would be nice to have a comment explaining that issue for future reference.